src/replayq.erl

```diff
@@ -515,13 +519,17 @@ ensure_deleted(Filename) ->
 %% The committer writes consumer's acked segmeng number + item ID
 %% to a file. The file is only read at start/restart.
 spawn_committer(ReaderSegno, Dir) ->
-  Name = iolist_to_binary(filename:join([Dir, committer])),
   %% register a name to avoid having two committers spawned for the same dir
```

Comment:
this was a lazy implementation to guard bugs in caller modules.
e.g. open without close.

we can maybe find a better way for this exclusiveness check.
e.g. put pid in app env, and assert if pid is alive.

Reply:
having a named process sounds like a simple implementation supported by the vm itself, with automatic cleanup of the name in case the committer terminates.

what would be the motivation to get rid of this mechanism?

Reply:
fixed in the latest commit